client/generator-carousel-admin/js/main.js

```diff
@@ -1,15 +1,14 @@
+/* eslint-env browser */
```

Comment:
This file is not transpiled hence it is still written in es5. If we make these changes we will lose some support.